include/bluetooth/mesh/vnd/le_pair_resp.h

```diff
@@ -35,19 +41,31 @@ extern "C" {
 /* @brief Invalidate previously used passkey.
  *
  * A user must call this function when a pairing request completes regardless of the status.
+ *
+ * @deprecated Use bt_mesh_le_pair_resp_passkey_set(INVALID_PASSKEY) instead.
  */
-void bt_mesh_le_pair_resp_passkey_invalidate(void);
+__deprecated void bt_mesh_le_pair_resp_passkey_invalidate(void);
 
 /* @brief Set own passkey instead of using randomly generating passkeys.
  *
  * By default, passkeys will be randomly generated on every new request. This function allows to use
  * pre-defined passkey instead.
  *
- * @params passkey Passkey to use for the pairing, or @ref BT_PASSKEY_INVALID to use randomly
+ * @params passkey Passkey to use for the pairing, or @ref INVALID_PASSKEY to use randomly
  *		   generated passkey again.
  */
 void bt_mesh_le_pair_resp_passkey_set(uint32_t passkey);
 
+/** @brief Get own passkey.
+ *
+ * This function will return the passkey set by the @ref bt_mesh_le_pair_resp_passkey_set function or it will return
+ * a randomly generated passkey by the le pair responder reset message and the passkey is invalidated immediately.
+ * @ref INVALID_PASSKEY is returned otherwise.
```

Comment:
```suggestion
 * This function will return the passkey set by the @ref bt_mesh_le_pair_resp_passkey_set function or it will return
 * a randomly generated passkey by the Reset message.
 * 
 * If the passkey has never been set or the Reset message has never been received, @ref BT_PASSKEY_RAND will be returned.
```